src/components/layout.tsx

```diff
@@ -9,7 +9,7 @@ import customMdx from '../components/customMdx'
 import './layout.css'
 import SidebarLayout from './sidebar'
 import TOC from './toc'
-import theme, { BaseStyle } from 'prisma-lens'
+import { LensProvider, theme } from '@prisma/lens/dist/web'
```

Comment:
I've made it so the LensProvider now loads up all fonts.

Web entrypoint: https://github.com/prisma/lens/pull/28/files#diff-248af8fc04398409c31555c8c23d55f57ee2c34fa828e420e66ed9ebbae08894R2
LensWebProvider: https://github.com/prisma/lens/pull/28/files#diff-1297fed6c78b9126feced3cc697d75b7451c87737dc558ef809583218d4ddef8R2
web.css: https://github.com/prisma/lens/pull/28/files#diff-07afb31730c4a9ae636a09776b94a855a9ff0bbad9037f1814c1dd3d4cfc87d2R1

This mirrors what happens with Lens components. Really it's just a component name change to the end user (like the docs!)